imx-atf fails to build with binutils 2.39

In current master of poky (081a3655fbb10b83793f102b975e153e75240682) where binutils is updated to 2.39 the package imx-atf fails to build for the imx8mp-lpddr4-evk machine (and most probably others to).

The failure is:
```
|   LD      <...>/imx-atf/2.6+gitAUTOINC+c6a19b1a35-r0/git/build/imx8mp/release/bl31/bl31.elf
| aarch64-poky-linux-ld: warning: <...>/imx-atf/2.6+gitAUTOINC+c6a19b1a35-r0/git/build/imx8mp/release/bl31/bl31.elf has a LOAD segment with RWX permissions
| make: *** [Makefile:1254: <...>/imx-atf/2.6+gitAUTOINC+c6a19b1a35-r0/git/build/imx8mp/release/bl31/bl31.elf] Error 1
```
A similar case in systemd with solution: https://github.com/systemd/systemd-stable/commit/8a6f966be404897b5333c218701965ac3b5a0806
Bug report for atf: https://developer.trustedfirmware.org/T996

Looking at the references above my guess is that a solution would be to add '-Wl,--no-warn-rwx-segments' but haven't been able to find where in the makefile structure to do it.

--- a/recipes-bsp/imx-atf/imx-atf_2.6.bb
+++ b/recipes-bsp/imx-atf/imx-atf_2.6.bb
@@ -29,7 +29,9 @@ EXTRA_OEMAKE += " \
 
 # Let the Makefile handle setting up the CFLAGS and LDFLAGS as it is a standalone application
 CFLAGS[unexport] = "1"
-LDFLAGS[unexport] = "1"
+# Needed for binutils >= 2.39 to prevent build failure.
+# imx-atf links with ld, thus no '-Wl,' prefix
+LDFLAGS = "--no-warn-rwx-segments"
 AS[unexport] = "1"
 LD[unexport] = "1"
 
@@ -56,7 +58,6 @@ EXTRA_OEMAKE += 'IMX_BOOT_UART_BASE=${ATF_BOOT_UART_BASE}'
 do_configure[noexec] = "1"
 
 do_compile() {
-    # Clear LDFLAGS to avoid the option -Wl recognize issue
     oe_runmake bl31
     if ${BUILD_OPTEE}; then
         oe_runmake clean BUILD_BASE=build-optee

